flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
```

Comment:
for now, i'd avoid pulling in dateutil here, i think, and look to factor this out either

https://pypi.python.org/pypi/equals looks vaguely similar. `equal`, `equality`, and `eql` are all 404 on PyPI so maybe those would work as names